src/_includes/soft-mod-cheat-sheet-banner.html

```diff
@@ -2,8 +2,7 @@
 {% assign background_position = 'top left' %}
 {% capture title %}{% t software-modernisation.cheat-sheet-cta-banner-title %}{% endcapture %}
 {% capture description %}{% t software-modernisation.cheat-sheet-cta-banner-text %}{% endcapture %}
-{% capture cta_href %}
-      https://info.codurance.com/en/case-for-change-questionnaire
-{% endcapture %}
+{% capture cta_href %}{% t software-modernisation.cheat-sheet-cta-href %}{% endcapture %}
 {% capture cta_text %}{% t software-modernisation.cheat-sheet-cta-banner-link %}{% endcapture %}
-{% include in_page_banner.html background_image=background_image background_position=background_position class="soft-mod-cheat-sheet-cta-banner" cta_href=cta_href cta_text=cta_text description=description title=title split="right-heavy" %}
+{% capture hubspot-cta %}{%- include soft_mod_hubspot_cheat_sheet_banner_cta.liquid -%}{% endcapture %}
```

Comment:
I think that the `hubspot-id` could be captured here instead of inside `soft_mod_hubspot_cheat_sheet_banner_cta.liquid`

The include within the `hubspot-cta` capture then just needs to be for the `hubspot-cta` component with the `hubspot-id` passed in.

Reply:
Done. This definitely makes it simpler :)